### PR TITLE
Fixing XIMEA parameters mapping (issue #4176)

### PR DESCRIPTION
**Pull request by SeninAndrew**
See http://code.opencv.org/issues/4176 for more details

### CODE REVIEW

File: modules/highgui/src/cap_ximea.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ class CvCaptureCAM_XIMEA : public CvCapture
 
     virtual bool open( int index );
     virtual void close();
-    virtual double getProperty(int);
+    virtual double getProperty(int) const;
```
Comment (alalek) [inline modules/highgui/src/cap_ximea.cpp]: Base class doesn't use `const` in 2.4: https://github.com/Itseez/opencv/blob/2.4/modules/highgui/src/precomp.hpp
Does it work really?

**Comment by SeninAndrew** [comment]
@alalek thanks a lot! I did not noticed the const difference between 2.4 and master - my fault. Hope it is okay now.

**Comment by SeninAndrew** [comment]
Hi @alalek, do you think all is good now? Thanks!

**Comment by SeninAndrew** [comment]
Hi @alalek. The fix to the master is already merged. Do you think we can merge this one to 2.4?

**Comment by alalek** [comment]
:+1:

**Pull request merged**